dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -479,7 +547,339 @@ private void moveAssetsToBackupDir() throws FileNotFoundException, IOException{
+						_bout = null;
+
+						break;
+					}
+
+					if(_list != null && _list.size() > 0 && _list.get(0) instanceof Comparable){
+						java.util.Collections.sort(_list);
+					}
+
+					_writing = new File(backupTempFilePath + "/" + clazz.getName() + "_" + formatter.format(i) + ".xml");
+					_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+
+					total = total + _list.size();
+
+					try {
+						Thread.sleep(10);
+					} catch (InterruptedException e) {
+						Logger.warn(this, "An error ocurred trying to create XML files");
+						Logger.error(this,e.getMessage(),e);
+					}
+
+					try {
+						_xstream.toXML(_list, _bout);
+					} finally {
+						_bout.close();
+					}
+
+					_bout = null;
+
+				}
+				Logger.info(this, "writing : " + total + " records for " + clazz.getName());
+			}
+
+
+			/* Run Liferay's Tables */
+			/* Companies */
+			_list = PublicCompanyFactory.getCompanies();
+			List<Company> companies = new ArrayList<Company>(_list);
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/" + Company.class.getName() + ".xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* Users */
+			_list = APILocator.getUserAPI().findAllUsers();
+			_list.add(APILocator.getUserAPI().getDefaultUser());
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/" + User.class.getName() + ".xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* users_roles */
+			dc = new DotConnect();
+
+			/* counter */
+			dc.setSQL("select * from counter");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Counter.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* counter */
+			dc.setSQL("select * from address");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Address.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollschoice */
+			dc.setSQL("select * from pollschoice");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Pollschoice.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollsdisplay */
+			dc.setSQL("select * from pollsdisplay");
+			_list = dc.getResults();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getResults"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)